vscode-lean4/src/utils/projectInfo.ts

```diff
@@ -2,15 +2,16 @@ import * as fs from 'fs';
 import { URL } from 'url';
 import { Uri, workspace, WorkspaceFolder } from 'vscode';
 import { addDefaultElanPath } from '../config';
+import { fsExistHelper, fsReadHelper } from './fsHelper';
 
 // Detect lean4 root directory (works for both lean4 repo and nightly distribution)
-export function isCoreLean4Directory(path: Uri): boolean {
-    return fs.existsSync(Uri.joinPath(path, 'LICENSE').fsPath) && fs.existsSync(Uri.joinPath(path, 'LICENSES').fsPath);
+export async function isCoreLean4Directory(path: Uri): Promise<boolean> {
```

Comment:
There is one more use of `isCoreLean4Directory` in `testLeanVersion` that you did not add an `await` keyword to.

Reply:
Ok good catch